docs/revanced-development/1_setup.md

```diff
@@ -28,19 +28,19 @@ A certain development environment is suggested to allow for streamlined developm
     
     ```bash
     for project in */; do
-        cd "$project" && ./gradlew build
+        cd "$project" && ./gradlew build && cd ..
```

Comment:
Why is .. necessary?

Line 30 should output full paths

Reply:
Not on my client, Git Bash for Windows. Here's the output of a test script:

```
for project in */; do
   echo "$project"
done
```
![image](https://github.com/user-attachments/assets/ad04406b-b597-49ef-8ba0-b653ab627694)

And so, only the first succeeds.